utils/docker/Dockerfile.el.8

```diff
@@ -13,74 +13,29 @@ LABEL maintainer="daos@daos.groups.io"
 # Intermittent cache-bust.  Used to reduce load on the actual CACHEBUST later.
 ARG CB0
 
-# If a local distro repository is supplied, then we should use it
-# and disable using the distro repositories that can be mirrored.
-# The Docker image starts out with the DISTRO provided GPG keys
-# on the disk, but not installed.  These must be installed before
-# adding any repo that needs GPG keys but does not provide meta
-# data for installing those keys.  Then the epel-release
-# package must be installed and the GPG key it provides must be
-# installed.
-ARG REPO_URL
-ARG REPO_DISTRO
-RUN if [ -n "$REPO_DISTRO" ]; then                              \
-      rpm --import /etc/pki/rpm-gpg/*;                          \
-      MY_REPO="${REPO_URL}${REPO_DISTRO}/";                     \
-      MY_NAME="${MY_REPO#*//}";                                 \
-      MY_NAME="${MY_NAME//\//_}";                               \
-      echo -e "[${MY_NAME}]\n\
-name=created from ${MY_REPO}\n\
-baseurl=${MY_REPO}\n\
-enabled=1\n\
-repo_gpgcheck=0\n\
-gpgcheck=1\n" >> /etc/yum.repos.d/local-el-group.repo;          \
-      dnf -y install --disablerepo extras --disablerepo baseos  \
-          epel-release-8-8.el8 dnf-plugins-core dnf-utils;      \
-      rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-8;         \
-      dnf config-manager --assumeyes --quiet --disable          \
-          baseos powertools epel epel-modular extras;           \
-      sed "s/^mirrorlist_expire=0*/mirrorlist_expire=99999999/" \
-          -i /etc/dnf/dnf.conf;                                 \
-      dnf -y upgrade epel-release;                              \
-    else                                                        \
-      dnf -y install dnf-plugins-core &&                        \
-      dnf config-manager --assumeyes --set-enabled powertools;  \
-      dnf -y install epel-release;                              \
-    fi;                                                         \
-    dnf config-manager --save --setopt=assumeyes=True &&        \
-    dnf config-manager --save --setopt=install_weak_deps=False && \
```

Comment:
These two were deliberate choices, by setting assumeyes=True here then the rest of the dockerfile can run without -y or --assumeyes which makes it shorter/clearer but also it means we can do the same with the install-el8.sh script which then means we can recommend it be run outside of docker to do the install without worrying about it making changes without prompting.  See the comment in install-el8.sh for details.

Reply:
In the effort to get the Dockerfiles in sync, this was probably removed.  Perhaps the effort of syncing should have went the other direction.  This PR is old enough now that I cannot remember why I chose the direction I did.

Reply:
The install_weak_deps=False option was an attempt to keep down the file size so we could take it or leave it but the decision to set assumeyes=True and not pass -y in the install-el8 script was a deliberate attempt to allow us to be able to tell users to invoke that script and have it behave as expected (i.e. they may be running it from another script or interactively and it would be correct in both cases)

Reply:
I can see a valid point in both of those.  Again, I don't recally why I chose to remove those from this file rather than propagate that change to other Dockerfiles.  I can re-spin this PR to do that if you wish.